charts/httpd/values.yaml

```diff
@@ -79,3 +79,10 @@ httpdConf:
 nodeSelector: {}
 tolerations: []
 affinity: {}
+httpdRestart:
+  enable: false # allow automatic kubernetes rollout restart
+  cron: '0 * * * *' # default to every hour
+  image:
+    repository: rancher/kubectl
+    tag: v1.28.13
```

Comment:
Since we use 1.29.x, WDYT about keeping this one by default, and then we'll had an updatecli manifest to track the version (based on the AKS version used on publick8s)?

Reply:
I planned to use the updatecli to update this right after this PR but you are right lets put directly the correct one.

Reply:
We're saying the same thing ;) We agree then